### DISCUSSION TITLE
Tree-sitter grammar fails to find npm parser module

### DISCUSSION THREAD

**Discussion by IHateYourCode** [atom/atom#23054] [Q&A]
Hi, I'm currently working on a grammer package,
I got my parser working and published the npm
package `@tomelang/porth`.

The package is set to public and I'm able to manually
install it via npm.

I wanted to start writing the grammar .cson and
prepared it so far, but Atom doesn't seem to be
able to find the npm module.

I even went as far as to publish the AtomIO module
in the hopes of the installation process automatically
finding and installing the parser packge, but no luck.

All related files are in the [repo](https://github.com/TomeLang/Porth)
parser branch for the parser,
master for the grammar (had to change it to master, `apm publish`
didn't work otherwise)

The error in question:
```
Cannot find module '@tomelang/porth'
Require stack:
- /home/user/.atom/packages/Porth/grammars/Porth.cson in /home/user/.atom/packages/Porth/grammars/Porth.cson
Hide Stack Trace
Error: Cannot find module '@tomelang/porth'
Require stack:
- /home/user/.atom/packages/Porth/grammars/Porth.cson
    at Module._resolveFilename (internal/modules/cjs/loader.js:797:17)
    at o._resolveFilename (electron/js2c/renderer_init.js:43:689)
    at Function.Module._resolveFilename (/usr/share/atom/resources/app.asar/src/module-cache.js:317:28)
    at new TreeSitterGrammar (/usr/share/atom/resources/app.asar/src/tree-sitter-grammar.js:55:39)
    at GrammarRegistry.createGrammar (/usr/share/atom/resources/app.asar/src/grammar-registry.js:604:14)
    at /usr/share/atom/resources/app.asar/src/grammar-registry.js:583:29
    at /usr/share/atom/resources/app.asar/node_modules/season/lib/cson.js:219:59
    at FSReqCallback.readFileAfterClose [as oncomplete] (internal/fs/read_file_context.js:61:3)
  at /home/user/.atom/packages/Porth/grammars/Porth.cson:1:1
```

I suspect maybe Atom, for some reason, doesn't
like parser packages sitting in npm organizations?

Any help appreciated.

**Comment by icecream17** [comment]
I'm not sure `apm publish` did the right thing here: https://atom.io/packages/porth

When you click on "repo" it shows
![image](https://user-images.githubusercontent.com/58114641/136099369-b41ae3a8-95fc-40d9-bf1b-1f5aa826daf1.png)

I think it's supposed to show
![image](https://user-images.githubusercontent.com/58114641/136099433-1ac4cc96-930a-4b46-992b-fdae2a714b1c.png)

Actually, `Documentation` is the default branch. Maybe change it to main
